brave/browser/brave_content_browser_client.cc

```diff
@@ -243,6 +243,10 @@ void BraveContentBrowserClient::ExposeInterfacesToRenderer(
       base::Bind(&SpellCheckHostImpl::Create, render_process_host->GetID()),
       ui_task_runner);
 #endif
+#if BUILDFLAG(HAS_SPELLCHECK_PANEL)
```

Comment:
@bridiver I believe this should be enclosed in the previous define (ENABLE_SPELLCHECK).